rllib/evaluation/sampler.py

```diff
@@ -92,7 +92,7 @@ def next(self) -> SampleBatchType:
         batches = [self.get_data()]
         batches.extend(self.get_extra_batches())
         if len(batches) > 1:
```

Comment:
I don't think you need this check anymore, Do you? What would concat_samples do if you have a list with only one item in it?

Reply:
Good catch, thanks!

Reply:
concat batches does not know wether it is normally handling MA or SA batches. To avoid abscure errors where we simply pass an empty single agent batch down the line when the sampler samples no batches at all, I think we should raise an error here. There are currently no cases of this happening I think.

Reply:
yes I think that's a good idea.